app/src/main/java/org/p2p/wallet/send/ui/search/adapter/SearchErrorViewHolder.kt

```diff
@@ -0,0 +1,69 @@
+package org.p2p.wallet.send.ui.search.adapter
+
+import androidx.recyclerview.widget.RecyclerView
+import android.view.ViewGroup
+import com.bumptech.glide.Glide
+import org.p2p.wallet.R
+import org.p2p.wallet.databinding.ItemSearchInvalidResultBinding
+import org.p2p.wallet.send.model.SearchResult
+import org.p2p.wallet.utils.CUT_ADDRESS_SYMBOLS_COUNT
+import org.p2p.wallet.utils.cutMiddle
+import org.p2p.wallet.utils.toPx
+import org.p2p.wallet.utils.viewbinding.inflateViewBinding
+import timber.log.Timber
+
+class SearchErrorViewHolder(
+    parent: ViewGroup,
+    private val binding: ItemSearchInvalidResultBinding = parent.inflateViewBinding(attachToRoot = false),
+) : RecyclerView.ViewHolder(binding.root) {
+
+    val iconPadding = 12.toPx()
+
+    fun onBind(item: SearchResult) {
+        with(binding) {
+            textViewAddress.text = item.addressState.address.cutMiddle(CUT_ADDRESS_SYMBOLS_COUNT)
+
+            when (item) {
+                is SearchResult.OwnAddressError -> {
+                    textViewDescription.setText(R.string.search_yourself_description)
+                    textViewError.setText(R.string.search_yourself_error)
+                    showWalletIcon()
+                }
+                is SearchResult.InvalidDirectAddress -> {
+                    val description = itemView.context.getString(
+                        R.string.search_no_other_tokens_description,
+                        item.directToken.symbol
+                    )
+                    textViewDescription.text = description
+                    textViewError.setText(R.string.search_no_other_tokens_error)
+                    loadTokenIcon(item.directToken.iconUrl)
+                }
+                // We are expecting no to get any other types of results for now
+                else -> {
+                    Timber.w("Unexpected search result type: $item")
```

Comment:
Timber.i

Reply:
I wouldn't say it's info message. 
That's the message to warn the dev about the problem and he should do smth

Reply:
it's would be logged same as Timber.i, fyi
if it's something that needs to be logged like a problem and have entry in Sentry - Timber.e

Reply:
However it will be logged. I just use the certain function for certain cases. 
Timber.w - `Log a warning message with optional format args.`
Timber.i - `Log an info message with optional format args.`

According to this, I assume that I want to log a warning message, not info. 
And I don't need to log Timber.e because it's not an error